apriltag_tracker_params:
  tag_size: 0.0858
  id: 0
  K: [413.3, 0, 381.44,  0.0, 412.78, 238.43, 0.0, 0.0, 1.0]
  D: [0.0,0.0,0.0,0.0]
  do_debug: false
